src/adapters/EasyRtcAdapter.js

```diff
@@ -54,22 +57,54 @@ class EasyRtcAdapter {
     this.easyrtc.setPeerListener(messageListener);
   }
 
+  updateTimeOffset() {
+    const clientSentTime = Date.now() + this.avgTimeOffset;
+
+    return fetch(document.location.href, { method: "HEAD", cache: "no-cache" })
```

Comment:
This is a temporary way to get some kind of server clock until we figure out an NTP based solution (or something similar) right?

Reply:
It's up to the adapter to decide how it wants to compute time offsets. This is a simple version that will work against any http server for the easyrtc adapter. We are free to do it however we like for our janus adapter.